src/api/axios.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
+    baseURL: 'http://localhost:8000/api/v1'
```

Comment:
Can we make the base part of this (`http://localhost:8000`) an environment variable? That way we can switch between the local version of the backend and the deployed version easily without adjusting the code

Reply:
done